filament/backend/src/metal/MetalDriver.mm

```diff
@@ -1008,11 +1015,6 @@
                                                                width:(srcTexture.width >> miplevel)
                                                               height:(srcTexture.height >> miplevel)
                                                            mipmapped:NO];
-#if defined(IOS)
```

Comment:
why was this removed?

Reply:
Because the default value is `Shared` on iOS and `Managed` on Catalyst and native macOS. The intent of the original author was to use `Shared` on iOS and `Managed` on native macOS. So instead of adding more macros to handle Catalyst I just removed the assignment altogether and let the default values persists. This is valid, because the descriptor itself was created a few lines above and the `storageMode` property is still on its default value.

Reply:
thanks